### PR TITLE
Show the ITF expiration date on the saved form page

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
See https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17032 for details.

I also added this to MVP since it was an easy addition.

## Testing done
Manually tested that it works.

### v2
image

### v1
image

## Acceptance criteria
- [x] The ITF expiration date is shown on the form saved page

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jbalboni — APPROVED** [review]

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cvalarida thank you.. a couple of fixes are needed: 

1. the incorrect header is on the screen for 526 All Claims. Can we update the header to for both applications to the below so it applies to both Increase and All Claims:  
**Your disability compensation application has been saved**

2. It doesn't look like the $ITF-expire-date is being pulled in, as the date in the message is earlier than the application saved date. It looks like in both instances the $ITF-established-date is being used. 

3. this part of the message still needs to be updated: 
If you’re on a public computer, please sign out of your account before you leave so your information stays secure.

**Comment by cvalarida** [comment]
@peggygannon 

1. Ahh, I see that now; I'll get that updated for the v2 form
2. It _is_ pulling in the `if.currentItf.expirationDate`--the expiration date is just weird on this one :man_shrugging: 
3. I can update this message for all the forms, but I'd rather not do it for just these forms specifically. Would that work?

**Comment by cvalarida** [comment]
@peggygannon Dang. So I've got a problem with 1. In the `FormSaved` component, I've got this:
```
Your {formTitles[formId]} application has been saved.
```
Because the v1 and v2 forms share the same `formId`, they share the same title. What do you think about changing the title for both so it says, "Your disability conversation application has been saved"?

**Comment by peggygannon** [comment]
@cvalarida Yes, that's fine for both V1 and V2 header to say: 
"Your disability compensation (21-526EZ) application has been saved"

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 1]
```diff
@@ -129,6 +130,7 @@ const formConfig = {
     noAuth:
       'Please sign in again to resume your application for disability claims increase.',
   },
+  formSavedPage: FormSavedPage,
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/config/form.js]: good to know; didn't know this hook existed actually.

File: src/applications/disability-benefits/all-claims/containers/FormSavedPage.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import FormSaved from '../../../../platform/forms/save-in-progress/FormSaved';
+
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+
+const FormSavedPage = props => {
+  const itfExpirationDate = moment(props.itf.currentITF.expirationDate).format(
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/containers/FormSavedPage.jsx]: it's not possible for someone to get the form into a state where it's saved without them also having an active ITF, right?

File: src/platform/forms/save-in-progress/helpers.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ export function createRoutesWithSaveInProgress(formConfig) {
   if (!formConfig.disableSave) {
     newRoutes.splice(newRoutes.length - 1, 0, {
       path: 'form-saved',
-      component: FormSaved,
+      component: formConfig.formSavedPage || FormSaved,
```
Comment (markgreenburg) [inline src/platform/forms/save-in-progress/helpers.js]: Oh 😛

**Review by markgreenburg — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/containers/FormSavedPage.jsx (hunk 2 above)
Comment (cvalarida) [reply src/applications/disability-benefits/all-claims/containers/FormSavedPage.jsx]: Technically yes? They'd have to be active on the form so their session doesn't expire by the time they reach their ITF expiration. In that event, though, it'd be "fine" as far as the data's concerned--that is, we won't be calling `currentITF of undefined`.

**Review by bshyong — APPROVED** [review]

**Comment by cvalarida** [comment]
@peggygannon How does this look?
image

**Comment by peggygannon** [comment]
@cvalarida thanks. This part of the message still needs to be updated. The updated content is this: 

If you’re on a public computer, please sign out of your account before you leave so your information stays secure.

**Comment by cvalarida** [comment]
@peggygannon Got it! It's been updated for all the forms.
image

**Review by bshyong — APPROVED** [review]

**Review by peggygannon — APPROVED** [review]

**Comment by peggygannon** [comment]
Thanks, @cvalarida

**Pull request merged**